mach

```diff
@@ -42,30 +43,45 @@ from tools.utils import (
 # === SUBCOMMANDS === #
 
 
-def _install(prefix=None, config=None):
-    configuration = "Debug"
-    if config:
-        configuration = config
-    cmake_cmd = ["cmake", "--install", "build", "--config", configuration]
+def _install(prefix=None, config="Release"):
+    cmake_cmd = ["cmake", "--install", "build", "--config", config]
     if prefix:
         cmake_cmd.extend(["--prefix", prefix])
     print(cmake_cmd)
-    subprocess.run(cmake_cmd, check=True)
+    if subprocess.call(cmake_cmd) == 1:
+        print("`cmake --install` returned with an error.")
+        if config == "Debug":
+            print(
+                "Please make sure to run `./mach build` before `./mach install --debug`."
+            )
+        else:
+            print(
+                "Please make sure to run `./mach build --release` before `./mach install`."
+            )
 
 
 @subcommand(
     [
         argument("-p", "--prefix", help="The path prefix to install into.", type=str),
+        argument("--debug", help="Install a debug build.", action="store_true"),
         argument(
-            "-c",
-            "--config",
-            help="The config to install, i.e. Debug or Release.",
-            type=str,
+            "--release", help="Install a release build. (default)", action="store_true"
```

Comment:
Looks like we don't actually use this. Let's drop it.

Reply:
Change in [0371b24](https://github.com/cryspen/hacl-packages/pull/249/commits/0371b24063dd71e562ad32edc3e245586758081c).